src/lib/browse/Filters.svelte

```diff
@@ -0,0 +1,127 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+  import { gjScheme } from "../../stores";
+  import CollapsibleCard from "../common/CollapsibleCard.svelte";
+  import FormElement from "../govuk/FormElement.svelte";
+  import SecondaryButton from "../govuk/SecondaryButton.svelte";
+  import Select from "../govuk/Select.svelte";
+  import { type Scheme } from "./data";
+
+  // These are immutable; re-create this component if they change
+  export let schemes: Map<string, Scheme>;
+
+  // by scheme_reference
+  export let showSchemes: Set<string> = new Set();
```

Comment:
Maybe `schemesToBeShown` would be a little more accurate

Reply:
Done